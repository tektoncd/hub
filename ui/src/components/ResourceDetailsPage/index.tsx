import React from 'react';
import { Grid, GridItem, PageSection } from '@patternfly/react-core';
import Details from '../../containers/Details';

const ResourceDetailsPage = () => {
  return (
    <PageSection>
      <Grid hasGutter>
        <GridItem span={12}>
          <Details />
        </GridItem>
      </Grid>
    </PageSection>
  );
};

export default ResourceDetailsPage;
