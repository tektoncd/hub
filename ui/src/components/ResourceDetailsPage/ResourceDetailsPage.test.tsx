import React from 'react';
import { shallow } from 'enzyme';
import ResourceDetailsPage from '.';

it('should render the resource details page  component', () => {
  const component = shallow(<ResourceDetailsPage />);
  expect(component.debug()).toMatchSnapshot();
});
