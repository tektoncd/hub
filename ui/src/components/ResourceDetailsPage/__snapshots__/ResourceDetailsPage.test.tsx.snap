// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the resource details page  component 1`] = `
"<PageSection>
  <Grid hasGutter={true}>
    <GridItem span={12}>
      <Memo() />
    </GridItem>
  </Grid>
</PageSection>"
`;
